eng/Versions.props

```diff
@@ -22,6 +22,6 @@
     <DotNetMonoManifestVersionBand>8.0.100</DotNetMonoManifestVersionBand>
     <DotNetEmscriptenManifestVersionBand>8.0.100</DotNetEmscriptenManifestVersionBand>
     <!-- NOTE: sometimes we hardcode this when transitioning to new version bands -->
-    <DotNetAndroidManifestVersionBand>$(DotNetPreviewVersionBand)</DotNetAndroidManifestVersionBand>
+    <DotNetAndroidManifestVersionBand>8.0.100</DotNetAndroidManifestVersionBand>
```

Comment:
I'm not seeing this used anywhere, should we update [Microsoft.NET.Sdk.Android.proj](https://github.com/xamarin/xamarin-android/blob/main/build-tools/create-packs/Microsoft.NET.Sdk.Android.proj#L13) to use this instead of `$(DotNetSdkManifestsFolder)`?

Reply:
Trying the other property now, (exactly what `release/7.0.4xx` has).

Reply:
This one seems to work:

![image](https://github.com/xamarin/xamarin-android/assets/840039/e678a912-7d52-4033-824d-765b2c28d148)

But letting CI finish.